### PR TITLE
APPS-935 Reduce the run time of docker build

### PR DESCRIPTION
**Pull request by edramaspk**

### CODE REVIEW

File: buildx-image-creation.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+TAG_LATEST=${TAG_LATEST:-false}
```
Comment (davi17g) [inline buildx-image-creation.sh:1]:
```suggestion
#!/bin/bash
TAG_LATEST=${TAG_LATEST:-false}
```

File: dockerfiles/create-abs-image.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+TAG_LATEST=${TAG_LATEST:-false}
```
Comment (davi17g) [inline dockerfiles/create-abs-image.sh:1]:
```suggestion
#!/bin/bash
TAG_LATEST=${TAG_LATEST:-false}
```

File: dockerfiles/create-base-image.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+docker buildx create --use --name mybuilder
```
Comment (davi17g) [inline dockerfiles/create-base-image.sh:1]:
```suggestion
#!/bin/bash
docker buildx create --use --name mybuilder
```

Comment (edramaspk) [reply dockerfiles/create-base-image.sh:2]: Done.

File: buildx-image-creation.sh
Comment (reugn) [inline buildx-image-creation.sh:1]: Why is this script required?

**Review by reugn — APPROVED** [review]

**Pull request merged**